definitions/procedures/service/base.rb

```diff
@@ -15,7 +15,7 @@ def self.common_params(context)
 
       def run_service_action(action, options)
         action_noun = feature(:service).action_noun(action).capitalize
-        puts "#{action_noun} the following service(s):\n"
+        puts "\n#{action_noun} the following service(s):"
```

Comment:
This spacing has bothered me for a while, I can do a separate PR to change, but its just a small thing so I included here! :cat: